docs/src/solutions.md

```diff
@@ -95,10 +95,12 @@ The container type (e.g., scalar, vector, or matrix) of the returned solution
 
 The objective value of a solved problem can be obtained via
 [`objective_value`](@ref). The best known bound on the optimal objective
-value can be obtained via [`objective_bound`](@ref).
+value can be obtained via [`objective_bound`](@ref). If dual values are
+available, the value of the dual objective can be obtained via
```

Comment:
Solvers could provide duals without providing a dual objective (e.g., OSQP), so this isn't precise.